<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="bgwidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(52, 101, 164, 255), stop:0.801047 rgba(255, 255, 255, 255));
border-color: rgb(85, 87, 83);
border: 2px solid;}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="2">
       <widget class="QLabel" name="labelT1">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Temp 1:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="writeT2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Ubuntu&quot;;
color: rgb(204, 0, 0);</string>
        </property>
        <property name="text">
         <string>write</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="writeT1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Ubuntu&quot;;
color: rgb(204, 0, 0);</string>
        </property>
        <property name="text">
         <string>write</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QCheckBox" name="checkSaveT3">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkSaveT2">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="writeT4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Ubuntu&quot;;
color: rgb(204, 0, 0);</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>write</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="valueT1">
        <property name="cursor">
         <cursorShape>ForbiddenCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 500 12pt &quot;Ubuntu&quot;;
border-color: rgb(0, 0, 0);
border: 2px solid;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="valueT2">
        <property name="cursor">
         <cursorShape>ForbiddenCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 500 12pt &quot;Ubuntu&quot;;
border-color: rgb(0, 0, 0);
border: 2px solid;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="writeT3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Ubuntu&quot;;
color: rgb(204, 0, 0);</string>
        </property>
        <property name="text">
         <string>write</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QCheckBox" name="checkSaveT4">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QLineEdit" name="valueT4">
        <property name="cursor">
         <cursorShape>ForbiddenCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 500 12pt &quot;Ubuntu&quot;;
border-color: rgb(0, 0, 0);
border: 2px solid;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QLineEdit" name="valueT3">
        <property name="cursor">
         <cursorShape>ForbiddenCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 500 12pt &quot;Ubuntu&quot;;
border-color: rgb(0, 0, 0);
border: 2px solid;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="labelT2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Temp 2:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QLabel" name="labelT4">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Temp 4:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="windowTitel">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>CO2 Messstand</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="labelT3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Temp 3:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="saveBtn">
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);
font: 12pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Start saving Data</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkSaveT1">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="8">
       <widget class="QWidget" name="plotWidget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>plotWidget</zorder>
     <zorder>windowTitel</zorder>
     <zorder>labelT1</zorder>
     <zorder>labelT2</zorder>
     <zorder>labelT3</zorder>
     <zorder>labelT4</zorder>
     <zorder>valueT1</zorder>
     <zorder>valueT2</zorder>
     <zorder>valueT3</zorder>
     <zorder>valueT4</zorder>
     <zorder>writeT1</zorder>
     <zorder>writeT2</zorder>
     <zorder>writeT3</zorder>
     <zorder>writeT4</zorder>
     <zorder>checkSaveT1</zorder>
     <zorder>checkSaveT2</zorder>
     <zorder>checkSaveT3</zorder>
     <zorder>checkSaveT4</zorder>
     <zorder>saveBtn</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
